packages/chart/src/FigureChartModel.ts

```diff
@@ -120,6 +129,12 @@ class FigureChartModel extends ChartModel {
     this.stopListeningFigure();
   }
 
+  getCachedChartUtils = memoize((dh: dhType) => new ChartUtils(dh), { max: 1 });
+
+  getChartUtils(): ChartUtils {
+    return this.getCachedChartUtils(this.dh);
+  }
+
```

Comment:
Why bother with memoization etc.? The dh is passed in once, and we'll never update it again... I'd just make it a member variable, e.g.

```
this.dh = dh;
this.chartUtils = new ChartUtils(dh);
```